treescript/src/treescript/mercurial.py

```diff
@@ -110,7 +110,7 @@ def build_hg_environment():
         config (dict): the running config.
 
     """
-    await run_hg_command(config, "-v", "version", return_output=True)
```

Comment:
Question: Out of curiosity, what did `return_output` do in this context?

Reply:
`run_hg_command` without that will print the results to stdout, so it ends up in the logs. With `return_output=True` it also returns it to the caller so it can be used - we were just throwing it away here